n_t/simulations.py

```diff
@@ -6,17 +6,14 @@
 from stdpopsim import homo_sapiens
 
 
-def homo_sapiens_Gutenkunst(path, seed, chrmStr, sample_size=20):
-    chrom = homo_sapiens.genome.chromosomes[chrmStr]
-    model = homo_sapiens.GutenkunstThreePopOutOfAfrica()
-    # model.debug()
-
-    # Currently sampling 20 individuals from a single popn.
+def simulate(out_path, species, model, genetic_map, seed, chrmStr, sample_size=20):
```

Comment:
We'll probably be replacing this with calls to the CLI in stdpopsim, so I don't think we need to worry about generalising too much here.

Reply:
Agreed, should we just keep it how it is for now?

Reply:
Yep, that's a plan.